src/components/Button/Button.ts

```diff
@@ -56,60 +55,36 @@ const REGULAR_BUTTON_ONLY_PROPS = [
   'stretch'
 ];
 
+enum SIZE_PROP_TYPE {
+  KILO,
+  MEGA,
+  GIGA
+}
+
+interface Props {
+  size: SIZE_PROP_TYPE // Size of the button. Use the Button's KILO, MEGA, or GIGA properties.
```

Comment:
The comments need to remain this format so they can be picked up by [react-docgen](https://github.com/reactjs/react-docgen).

```suggestion
  /**
   * Size of the button. Use the Button's KILO, MEGA, or GIGA properties.
   */
  size: SIZE_PROP_TYPE 
```